src/lib/components/profile/modal/DeleteModal.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+  import Modal from './Modal.svelte'
+
+  export let title: string
+  export let isOpen = false
+  let disabled = false
+
+  function closeModal() {
+    isOpen = false
+  }
+
+  export let handleDelete: () => Promise<void> | void
```

Comment:
O soy void o te prometo que seré void 😶

Reply:
```suggestion
  export let handleDelete: () => Promise<void>
```
Creo que es mejor reducirlo a esto y wrapear el void en una promesa, si no es tan complicado de implementar